src/modals/ConnectModal.svelte

```diff
@@ -0,0 +1,197 @@
+  import IconSmall from "@app/components/IconSmall.svelte";
+  import Link from "@app/components/Link.svelte";
+  import TextInput from "@app/components/TextInput.svelte";
+
+  $: customUrl = `${httpd.api.baseUrl.scheme}://${httpd.api.baseUrl.hostname}:${customPort}`;
+  $: command = import.meta.env.PROD
+    ? `rad web --backend ${customUrl}`
+    : `rad web --frontend ${
+        new URL(import.meta.url).origin
+      } --backend ${customUrl}`;
+  let customPort = httpd.api.port;
+  $: validPortNumber = Number(customPort) > 0 && Number(customPort) <= 65535;
+
+  $: if ($httpdStore.state === "authenticated") {
+    modal.hide();
+  }
+</script>
+
+<style>
+  .container {
+    display: flex;
+    flex-direction: column;
+    width: 33rem;
+    margin-top: 1.5rem;
+    gap: 1.5rem;
+  }
+  .progress {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+  }
+  .progress-bar {
+    height: 6px;
+    border-radius: var(--border-radius-round);
+    background-color: var(--color-background-dip);
+  }
+  .bar {
+    display: flex;
+    background-color: var(--color-fill-secondary);
+    height: 100%;
+    border-radius: var(--border-radius-round);
+  }
+  .captions {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    color: var(--color-foreground-dim);
+    font-size: var(--font-size-tiny);
+  }
+
+  .input {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    gap: 0.75rem;
+  }
+
+  .status {
+    font-size: var(--font-size-tiny);
+    color: var(--color-fill-gray);
+    text-align: left;
+  }
+  .separator {
+    height: 1px;
+    background-color: var(--color-border-hint);
+  }
+
+  .host {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: var(--font-size-small);
+  }
+
+  .label {
+    text-align: left;
+    font-size: var(--font-size-small);
+  }
+</style>
+
+<FullscreenModal title="Connect and authenticate" closeAction={false}>
+  <Icon name="review" size="48" slot="icon" />
+
+  <svelte:fragment slot="subtitle">
+    Complete the steps below to browse projects on your local machine,
+    <br />
+    create issues, and participate in discussions.
+  </svelte:fragment>
+
+  <div class="container" slot="body">
+    {#if $httpdStore.state === "stopped"}
+      <div class="progress">
+        <div class="progress-bar">
+          <div class="bar" style:width="1%" />
+        </div>
+        <div class="captions">
+          <div
+            style:text-align="left"
+            style:color="var(--color-fill-secondary)">
+            Start httpd server
+          </div>
+          <div>Authenticate</div>
+          <div style:text-align="right">Done</div>
+        </div>
+      </div>
+
+      <div class="input">
+        <div class="label">
+          Run this command in your terminal to connect to your local node:
+        </div>
+        <Command fullWidth command="radicle-httpd" />
```

Comment:
Hm, that works only if the command contents fill the width of the modal.

This is what we get when the command is short:
<img width="813" alt="Screenshot 2023-10-11 at 19 11 58" src="https://github.com/radicle-dev/radicle-interface/assets/158411/639f7ebd-2afc-4d78-bba2-220d75c7429e">

With a long command your approach works:
<img width="712" alt="Screenshot 2023-10-11 at 19 12 35" src="https://github.com/radicle-dev/radicle-interface/assets/158411/74cd231a-9bc5-4d1e-a223-4d9935fd5273">

And here's our design requirement:
<img width="420" alt="Screenshot 2023-10-11 at 19 12 04" src="https://github.com/radicle-dev/radicle-interface/assets/158411/6d2aa9e6-080d-4997-ac51-e042ac0a374c">

Reply:
> Hm, that works only if the command contents fill the width of the modal.

If you also get rid of the `.wrapper` div in `Command` it works.

Reply:
I tried your suggestion, but then we'd also have to change all the other 5 call sites of the `<Command>` component to include a wrapper around it. Otherwise we get this.

<img width="1840" alt="Screenshot 2023-10-13 at 18 03 32" src="https://github.com/radicle-dev/radicle-interface/assets/158411/2238d321-7c83-433c-933a-cd1cb355373e">